Zend/tests/array_offset.phpt

```diff
@@ -13,13 +13,13 @@ echo "Done\n";
 --EXPECTF--
 Warning: Undefined array key -1 in %s on line %d
 
-Deprecated: Implicit conversion from non-compatible float -1.1 to int in %s on line %d
+Deprecated: Implicit conversion from incompatible float -1.1 to int in %s on line %d
```

Comment:
As the bikeshed is open now: I think "incompatible float" is still not super meaningful to the user. Maybe we could do something like:

```suggestion
Deprecated: Implicit conversion from float -1.1 to int drops fraction in %s on line %d
```

Though that will not work for very larger floats where it's not the fraction that gets lost. So maybe:

```suggestion
Deprecated: Implicit conversion from float -1.1 to int loses precision in %s on line %d
```

Or something similar?

Reply:
I think the "loses precision" variant is the best, even if it doesn't convey conversions from NAN,INF, and out of range floats, but this might be acceptable for the general case

Reply:
Thank you, I changed it to `loses precision` version.